README.md

```diff
@@ -1,160 +1,132 @@
-# Privacy Pass Extension
-
+[![GitHub release](https://img.shields.io/github/release/privacypass/challenge-bypass-extension.svg)](https://GitHub.com/privacypass/challenge-bypass-extension/releases/)
 [![Privacy Pass](https://github.com/privacypass/challenge-bypass-extension/actions/workflows/action.yml/badge.svg)](https://github.com/privacypass/challenge-bypass-extension/actions)
+[![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
+<h1 align="center">Privacy Pass Extension</h1>
+<p align="center">
+<img src="./public/images/gold-badge.svg" width="256" height="256" />
+</p>
+
+This browser extension implements the client-side of the Privacy Pass protocol. It provides unlinkable cryptographic tokens that can be used for bypassing CAPTCHAs.
```

Comment:
```suggestion
This browser extension implements the client-side of the Privacy Pass protocol. It provides unlinkable cryptographic tokens. For example, these can be used on Cloudflare-protected websites to redeem a token instead of having to solve a  CAPTCHAs.
```

modify the description to be closer to the [homepage](https://privacypass.github.io/) mentioned thereafter. Mainly, there is no bypass, and it's not CAPTCHA specific.